packages/commonwealth/client/scripts/App.tsx

```diff
@@ -17,6 +20,8 @@ const Splash = () => {
   );
 };
 
+OpenFeature.setProvider(new InMemoryProvider(featureFlags));
```

Comment:
guess this is the place you'll use to decide adapter based on NODE_ENV?

Reply:
Yes. This will be changed in the followup PR where on prod we will be using the Unleash provider.